### PR TITLE
Add documentation for cloning this repo

### PR DESCRIPTION
**Pull request by mbtools**
Provides details on how to clone this repo using abapGit for students and contributors

### CODE REVIEW

File: docs/CLONE.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Clone the ABAP Track
+
+## Prerequisites
+
+1. ABAP system release 7.4 or higher
+3. abapGit v1.115.0 or higher
+2. Authorizations to implement ABAP packages, programs, and classes
```
Comment (schmelto) [inline docs/CLONE.md:7]:
```suggestion
2. abapGit v1.115.0 or higher
3. Authorizations to implement ABAP packages, programs, and classes
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,17 +34,17 @@ The baseline of work is as follows:
...
-2. Implement unit tests for your ABAP class
-3. Stage and commit the SAP package and class 
-4. To create an example solution, we recommend the following approach since you can't use abapGit for it:
-- Copy class to `ZCL_<EXERCISE>` to `ZCL_<EXERCISE>_EXAMPLE` *and* put it into SAP package `$TMP` (or another package outside the repo)
-- Implement the solution in `ZCL_<EXERCISE>_EXAMPLE`
-- Add the class code and unit tests (testclasses) to the `.meta` directory of your branch
-7. Create an entry in `config.json`: a unique _new_ UUID (you can use the `configlet uuid` tool to generate one, scroll down to [tools](#tools) to see how you can get it), give it a difficulty (should be similar to similar exercises), and make sure the _order_ of the file is sane.
+1. Implement unit tests for your ABAP class
+1. Stage and commit the SAP package and class 
+1. To create an example solution, we recommend the following approach since you can't use abapGit for it:
```
Comment (schmelto) [inline CONTRIBUTING.md:43]: Don't know of the numeration here will be an issue

Comment (mbtools) [reply CONTRIBUTING.md:43]: Markdown handles this automatically (see preview)

https://github.com/exercism/abap/blob/7e6e1d69eeb4655194765ea8aedae37ab4c6c395/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,17 +34,17 @@ The baseline of work is as follows:
 
 1. Open a new issue, we'll label it with `new exercise ✨`
 1. We'll assign the issue to you, so you get to work on this exercise
-1. Clone this repository to your own SAP system using abapGit. We recommend to use SAP package `$EXECISM`
+1. Clone this repository to your own SAP system using [abapGit](https://github.com/abapGit/abapGit). We recommend to use SAP package `$EXECISM`
```
Comment (g-back) [inline CONTRIBUTING.md:37]:
```suggestion
1. Clone this repository to your own SAP system using [abapGit](https://github.com/abapGit/abapGit). We recommend to use SAP package `$EXERCISM`
```

Comment (mbtools) [reply CONTRIBUTING.md]: good catch Thank you

**Pull request merged**